contributing.md

```diff
@@ -13,18 +13,33 @@ machine to develop Bazel and, when you've made a patch, how to submit it.</p>
 In general, we prefer contributions that fix bugs or add features (as opposed to
 stylistic, refactoring, or "cleanup" changes).
 
-Please check with us on the [dev
-list](https://groups.google.com/forum/#!forum/bazel-dev) before investing a lot
-of time in a patch. Meet Bazel maintainers and other contributors on
-[Slack](https://slack.bazel.build).
+### What issues can I work on?
+
+In general, if there is a Bazel team member assigned to an issue, someone is
+working on it. If not, you can use the priority level to as a rough guide.
+
+-   **P0**: someone from the Bazel team is be urgently working on it. If it is
```

Comment:
might not necessarily be someone on the Bazel team, even for P0? Although it'll be good to co-assign someone from the bazel team if remediation is performed by someone from the community.